CHANGELOG.md

```diff
@@ -6,6 +6,32 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [2.4.0]
+### Uncategorized
```

Comment:
These need to be categorized into "Fixed", "Changed" or "Added" to pass the linter

And we can remove the "chore" entry entirely